### PR TITLE
Map string offsets to columns to ensure proper tab indentation.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/38819

### CODE REVIEW

File: src/Workspaces/Core/Portable/Indentation/AbstractIndentationService.Indenter.cs [hunk 1]
```diff
@@ -126,12 +126,25 @@ public bool TryGetSmartTokenIndentation(out IndentationResult indentationResult)
                     if (LineToBeIndented.LineNumber < updatedSourceText.Lines.Count)
                     {
                         var updatedLine = updatedSourceText.Lines[LineToBeIndented.LineNumber];
-                        var offset = updatedLine.GetFirstNonWhitespaceOffset();
-                        if (offset != null)
+                        var nonWhitespaceOffset = updatedLine.GetFirstNonWhitespaceOffset();
+                        if (nonWhitespaceOffset != null)
                         {
+                            // 'nonWhitespaceOffset' will simply an int indicating how many
```
Comment (CyrusNajmabadi) [inline src/Workspaces/Core/Portable/Indentation/AbstractIndentationService.Indenter.cs]:
```suggestion
                            // 'nonWhitespaceOffset' is simply an int indicating how many
```

**Comment by CyrusNajmabadi** [comment]
@sharwell please review this small change when convenient.

**Comment by CyrusNajmabadi** [comment]
Thanks.  Feel free to merge in. @sharwell   :)

**Comment by sharwell** [comment]
Blocked pending servicing bar check

**Comment by sharwell** [comment]
Superseded by #38893

**Pull request closed without merging**